### PR TITLE
feat: add archon shards data

### PR DESCRIPTION
**Pull request by SlayerOrnstein**
### What did you fix? 
Adds archon shard data including upgrade types. Thanks @Sainan 

---

### Reproduction steps

---

### Considerations
- Does this contain a new dependency? **No**
- Does this introduce opinionated data formatting or manual data entry? **Yes**
- Does this pr include updated data files in a separate commit that can be reverted for a clean code-only pr? **Yes**
- Have I run the linter? **Yes**
- Is is a bug fix, feature request, or enhancement? **Feature**

### CODE REVIEW

**Comment by SlayerOrnstein** [comment]
@TobiTenno I copied and pasted the same code from the test that I added to the translation bit but commented it out because there are no translations yet

File: test/utilities/translation.spec.js [hunk 1]
```diff
@@ -493,5 +514,24 @@ describe('translation', () => {
...
+    //   it("should return a translation of the key if it's found in the data", () => {
+    //     archonShardUpgradeType(
+    //       'ACC_BLUE',
+    //       '/Lotus/Upgrades/Invigorations/ArchonCrystalUpgrades/ArchonCrystalUpgradeWarframeHealthMax'
+    //     ).should.equal('+150% Health');
+    //   });
+    //   it("should return the key if it's not found in the data", () => {
+    //     archonShardUpgradeType('notfound', 'notfound').should.equal('notfound');
+    //   });
+    // });
```
Comment (TobiTenno) [inline test/utilities/translation.spec.js:535]: should be able to comment these and they should still work, that's how the fallback files work

**Review by TobiTenno — CHANGES_REQUESTED** [review]
uncomment the test, it should work if it would work w/ translation files

**Review by TobiTenno — APPROVED** [review]

**Pull request merged**